### ISSUE TITLE
CONTRIBUTING.md references Meta CLA

### ISSUE THREAD

**Issue by littledan** [TimDettmers/bitsandbytes#1198]
### System Info

(GitHub)

### Reproduction

In https://github.com/TimDettmers/bitsandbytes/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla , there is the following:

> ## Contributor License Agreement ("CLA")
>
> In order to accept your pull request, we need you to submit a CLA. You only need to do this once to work on any of Facebook's open source projects.
>
> Complete your CLA here: https://code.facebook.com/cla

The use of Facebook's CLA is hard to understand, given that this is a personal repo and no CLA bot seems to be used.

### Expected behavior

Facebook's CLA should not be used.

**Comment by Titus-von-Koeller** [comment]
Yes, this is right, it's still a left over from before. I'll remove it
